mudlet-mapper.xml

```diff
@@ -4672,6 +4678,28 @@ end</script>
       mmp.clearpathcache() -- clear cache so mmp.getPath accounts for the new way
       madeduanathar = true
     end
+
+	-- Stratospheric Harness support -- PapaGuacamole
+	elseif mmp.game == &quot;achaea&quot; and mmp.settings.harness and not table.contains(gmcp.Room.Info.details, &quot;indoors&quot;)
+	  and (mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Eastern_Isles&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Northern_Isles&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Western_Isles&quot;))
```

Comment:
If you are checking for the continent later on anyways, you could leave this part of the condition out.